index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
+      <div id="title">
+        <h1>Dr. Norman Borlaug</h1>
+        <p>the man who saved a billion lives</p>
+      </div>
+      <div id="img-div">
+        <img id = "image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="Dr noman">
+        <div id="img-caption">
+          <p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>
+        </div>
+      </div>
+      <section id="tribute-info">
+        <h1>Here's the line of Dr. Borlaug's life</h1>
+        <ul>
+          <li>
+            <strong>1914</strong> -Born in Cresco, Iowa</li>
+          
+          <li><strong>1933</strong> - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"</li>
+         
+          <li><strong>1935</strong>- Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."</li>
+          
+          <li><strong>1937</strong>  - Finishes university and takes a job in the US Forestry Service</li>
+        
+        </ul>
+       
+
+      <em>"Borlaug's life and achievement are testimony to the far-reaching contribution<br> that one man's towering intellect, persistence and scientific vision can make to <br>human peace and progress."<br><br>
+
+        -- Indian Prime Minister Manmohan Singh</em>
+      </section>
+      <p>If you have time, you should read more about this incredible human being on his <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">wikipedia entry
+      </a>
+
+
+
+
+    </main>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
- [ ] Good job explaining "why" you are asking for a change but I think it could be a good idea to make suggestions instead of giving orders to make the communication more friendly. You could add some emojis to achieve this and make the communication more human :wink:

Reply:
- [ ] Good job explaining "why" you are asking for changes but I think it could be a good idea to make suggestions instead of giving orders, this will make the communication more friendly. You could add some emojis to make the communication more human :wink: